docs/usage/edsl_examples.rst

```diff
@@ -314,15 +326,270 @@ coefficients, and ``P`` gives the padding offsets.
+
+    .. literalinclude:: ../../plaidml/edsl/examples/gemv.py
+      :pyobject: gemv2
+
+GEMM BLAS Level 3
+*******************
+This example illustrates matrix-matrix operations using the generalized matrix-matrix (GEMM) multiplication of the form:
+
+
+.. math::
+
+   \color{red}O
+   \color{default}  =
+   \color{turquoise} \alpha
+   \color{blue}A
+   \color{orange} *
+   \color{blue} B
+   \color{green} +
+   \color{turquoise} \beta
+   \color{blue}C
+
+
+Here :math:`\color{blue}A` is a matrix, :math:`\color{purple}x` and :math:`\color{purple}y` 
+are vectors and :math:`\color{turquoise}\alpha` and  :math:`\color{turquoise}\beta` are 
+constants. 
+
+We can represent the matrix operation involved as:
+
+.. math::
+   {\color{red}O[i, j]}
+   \color{default}  =
+   \color{green} \sum_{k} (
+   \color{turquoise} \alpha
+   \color{magenta}\cdot
+   \color{blue}A[
+   \color{default}i, k
+   \color{blue}]
+   \color{orange} *
+   \color{blue} B[
+   \color{default}k, j
+   \color{blue}] 
+   \color{green} )
+   \color{magenta} +
+   \color{turquoise} \beta
+   \color{magenta}\cdot
+   \color{blue}C[
+   \color{default}i, j
+   \color{blue}]
+
+
+This can easily be written in eDSL as follows. 
+
+
+.. math::
+
+    \begin{aligned}
+    &\color{blue}\verb! A!
+    \color{default}\verb! =!
+    \color{blue}\verb! A!
+    \color{magenta}\verb! *!
+    \color{turquoise}\verb! alpha! \\
+    &\color{blue}\verb! C!
+    \color{default}\verb! =!
+    \color{blue}\verb! C!
+    \color{magenta}\verb! *!
+    \color{turquoise}\verb! beta! \\
+    \verb!Contraction()!
+    &\verb!.outShape(I,J)!\\
+    &{\color{red}\verb!.outAccess(i,j)!}\\
+    &{\color{green}\verb!.sum(!}\\
+    &{\color{blue}\verb!        A(!}
+    \verb!i,k!
+    {\color{blue}\verb!)!}
+    {\color{orange}\verb!  *  !}
+    {\color{blue}\verb!B(!}
+    \verb!k,j!
+    {\color{blue}\verb!)!}\\
+    &{\color{green}\verb!     )!}\\
+    &{\color{magenta}\verb!+!}
+    {\color{blue}\verb!  C!}\\
+    \end{aligned}
+
+Where the contraction :math:`{\color{green}\verb!sum(...)!}`
+handles the vector multiplication :math:`\color{blue}A \color{orange}* \color{blue}B`, the element wise operation 
+:math:`{\color{magenta}\verb!+!}` handles the vector addition :math:`\color{green}(...)\color{magenta}+ \color{blue}C`.
+The constant-vector multiplication is handled by an element wise :math:`{\color{magenta}\verb!*!}` operator. 
+
+.. tabs::
+
+  .. group-tab:: C++
+
+    .. literalinclude:: ../../plaidml/edsl/examples/gemm.cc
+      :language: cpp
+      :start-after: gemm_start
+      :end-before: gemm_end
+
+  .. group-tab:: Python
+
+    .. literalinclude:: ../../plaidml/edsl/examples/gemm.py
+      :pyobject: gemm
+
+
+Uniform 8-bit Quantization
+*******************
+
+Quantization is a promising new approach used to reduce memory and compute resources required by neural network operations.
+This examples illustrates 8-bit uniform quantization operation :cite:`ob2018quantization` in eDSL. 
```

Comment:
The citation here is not rendering correctly on my machine, but perhaps I'm building it wrong.

Reply:
Thanks Tim! There was a typo. I have fixed it and also changed the style to a cleaner numbered one. See if this seems better.

Reply:
Looks good now, thanks for fixing!